.changeset/polite-gifts-join.md

```diff
@@ -0,0 +1,5 @@
+---
+"@chakra-ui/utils": minor
```

Comment:
I'm going to change this to `patch` since it's only a fix. `minor` would be for adding a new feature.